packaging/flavorgen/cmd/root.go

```diff
@@ -56,6 +57,19 @@ func RunRoot(command *cobra.Command) error {
 		util.PrintObjects(flavors.MultiNodeTemplateWithKubeVIP())
 	case "external-loadbalancer":
 		util.PrintObjects(flavors.MultiNodeTemplateWithExternalLoadBalancer())
+	case "cluster-class":
+		util.PrintObjects(flavors.ClusterClassTemplateWithKubeVIP())
+	case "cluster-topology":
+		additionalReplacements := []util.Replacement{
+			{
+				Kind:      "Cluster",
+				Name:      "${CLUSTER_NAME}",
+				Value:     env.ControlPlaneMachineCountVar,
+				FieldPath: []string{"spec", "topology", "controlPlane", "replicas"},
+			},
+		}
```

Comment:
There should be a replacement for the number of worker nodes too.

Reply:
Thanks for helping with this issue over Zoom! :) 
I will use `SetNestedSlice` like we talked about to replace the number of worker nodes.